.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,34 +6,31 @@ Assignees: "One Platform Developers"
 Projects: "One Platform Development"
 -->
 
+# Closes/Fixes/Resolves
 
-> ### Closes/Fixes/Resolves #issue_id
+<!-- #issue_id -->
```

Comment:
A keyword is needed before the issue_id to ensure proper link between the issue and the pr.
(See https://github.blog/2013-05-14-closing-issues-via-pull-requests/)

```suggestion
> ### <!-- Any of the keywords: Closes/Fixes/Resolves followed by #issue_id (should be separated by a space only) -->
```